src/lib.rs

```diff
@@ -31,7 +31,11 @@ pub(crate) static GLOBAL_ALERT: Option<GlobalAlert> = Some(GlobalAlert {
     url: "https://blog.rust-lang.org/2019/09/18/upcoming-docsrs-changes.html",
     text: "Upcoming docs.rs breaking changes!",
     css_class: "error",
+<<<<<<< HEAD
     fa_icon: "warning",
+=======
+    fa_icon: "exclamation-triangle",
+>>>>>>> 6e25e2c... Pause for upstream
```

Comment:
Please fix this.

Reply:
No idea how I missed these, but they're fixed now